app/src/main/java/com/example/solanakotlincomposescaffold/viewmodel/MainViewModel.kt

```diff
@@ -35,7 +40,8 @@ class MainViewModel @Inject constructor(
     private val persistenceUseCase: PersistenceUseCase
 ): ViewModel() {
 
-    private val rpcUri = "https://api.testnet.solana.com".toUri()
+    private val rpcUri = "https://api.devnet.solana.com".toUri()
```

Comment:
nit: best to provide this through `BuildConfig` or other mechanism where it can be injected easily